JSIL/Transforms/ReplaceMethodCalls.cs

```diff
@@ -314,7 +314,7 @@ public class ReplaceMethodCalls : JSAstVisitor {
                             return;
                         }
                     }
-                } else if (
+                } /*else if (
```

Comment:
mind killing the commented-out code instead? Seems like it isn't needed anymore.